jobs/config.json

```diff
@@ -219,9 +219,18 @@
   },
   "ci-ingress-gce-upgrade-e2e": {
     "args": [
-      "true"
+      "--check-leaked-resources",
+      "--cluster=",
+      "--extract=ci/latest",
+      "--extract=ci/latest",
```

Comment:
I doubt this will work... and probably the upgrade job change can be in a separate PR?